src/main/java/recipeio/commands/FindCommand.java

```diff
@@ -0,0 +1,84 @@
+package recipeio.commands;
+
+import recipeio.recipe.Recipe;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+
+public class FindCommand {
+    public static void findKeyword(String keyword, ArrayList<Recipe> recipes) {
+        ArrayList<Recipe> matches = new ArrayList<>();
+        if (recipes.isEmpty()) {
+            System.out.println("Sorry, you have no recipes to find matches with. Try adding some!");
+        } else {
```

Comment:
Hi Nidhi, I think you can return out from the if block, and start the main code without using an 'else'. this will help in preventing arrow-head code.

```suggestion
        if (recipes.isEmpty()) {
            System.out.println("Sorry, you have no recipes to find matches with. Try adding some!");
            return;
        }
        // main code here
```